packages/react-native-bpk-appearance/src/common-types.js

```diff
@@ -24,13 +24,13 @@ export type BpkDynamicValue<T> = { light: T, dark: T };
 // prettier does't handle this well
 // prettier-ignore
 export type UnpackedBpkDynamicValue<X> = $Call<
-  & ((BpkDynamicValue<string>) => string)
-  & ((BpkDynamicValue<number>) => number)
-  & ((BpkDynamicValue<boolean>) => boolean)
-  & ((BpkDynamicValue<Array<string>>) => Array<string>)
-  & ((BpkDynamicValue<Array<number>>) => Array<number>)
-  & ((BpkDynamicValue<Array<boolean>>) => Array<boolean>)
-  & ((BpkDynamicValue<*>) => any)
+  & ((string | BpkDynamicValue<string>) => string)
```

Comment:
Surely `BpkDynamicValue<string>` includes `string`?
If not shouldn't we change `BpkDynamicValue` instead of every usage of it?